haiku-apps/yab_buildfactory/yab_buildfactory-2.4.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="A buildfactory for yab programs"
+DESCRIPTION="The buildfactory allows you to generate standalone programs."
+HOMEPAGE="https://software.besly.de/"
+COPYRIGHT="2006-2015 Jan Bungeroth
+	2015-2017 Jim Saxton
+	2020 Lorenz (lorglas) BeSly Software Solutions"
+LICENSE="Artistic"
+REVISION="1"
+SOURCE_URI="https://github.com/lorglas/yab_buildfactory/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="c1bf29537fcb1a6988ca81bcad1485532090ddd5f11fe962a68c40f927822489"
+SOURCE_FILENAME="yab_buildfactory-$portVersion.tar.gz"
+ADDITIONAL_FILES="yab_buildfactory.sh
+	yab.hvif"
+
+ARCHITECTURES="x86_gcc2 !x86 x86_64"
+DISABLE_SOURCE_PACKAGE="YES"
+
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/yab_buildfactory.sh"
+
+PROVIDES="
+	yab_buildfactory = $portVersion
+	app:yab_buildfactory = $portVersion
+	"
+REQUIRES="
+	haiku
+	yab
+	devel:libz
+	"
+#devel:libz is needed both to build and for the BuildFactory to do its work.
+
+BUILD_REQUIRES=""
+BUILD_PREREQUIRES=""
+
+INSTALL()
+{
+	mkdir -p $appsDir/yab-buildfactory
+	addattr -t icon -f $portDir/additional-files/yab.hvif "BEOS:ICON" "BeSly_Buildfactory.yab"
```

Comment:
There is no aditional-files directory in yab_buildfactory-$portVersion.tar.gz-$portVersion.tar.gz

Reply:
Additional-files are supplied in this PR (hence $portDir/additional-files)